### PR TITLE
add all icons, categories, subcategories to catagories.yml

### PR DESCRIPTION
**Pull request by chrisconnors-ibm**
completed the categories.yml file based on organization from @conradennis 

#### Changelog

**Changed**

- All categories, sub categories, and their members added; known future icon names present but commented out

### CODE REVIEW

File: packages/icons/categories.yml [hunk 1]
```diff
@@ -1,25 +1,764 @@
...
+      - compare
+      - connect
+      - deploy-rules
+      - drag--horizontal
+      - drag--vertical
+      - fade
+      - filter
+      - filter--edit
+      - flag
+      # fork
```
Comment (shixiedesign) [inline packages/icons/categories.yml:155]: Is this # an error? Should this just be `- fork` ?

File: packages/icons/categories.yml [hunk 2]
```diff
@@ -1,25 +1,764 @@
...
+      - contrast
+      - corner
+      - crop
+      - cut
+      - draw
+      - edit
+      - erase
+      - eyedropper
+      - image
+      # language
```
Comment (shixiedesign) [inline packages/icons/categories.yml:63]: Is the `#` supposed to be `-` ?

File: packages/icons/categories.yml [hunk 3]
```diff
@@ -1,25 +1,764 @@
...
+      - wireless-checkout
+    - subcategory: Health
+      members:
+      - 3D-cursor
+      - 3D-curve--auto-colon
+      - 3D-curve--auto-vessels
+      - 3D-curve--manual
+      - 3D-MPR-toggle
+      - 3rd-party-connected
+      # 3D-software
```
Comment (shixiedesign) [inline packages/icons/categories.yml:310]: another # here

File: packages/icons/categories.yml [hunk 4]
```diff
@@ -1,25 +1,764 @@
...
+      - page-scroll
+      - pet-image--B
+      - pet-image--O
+      - pills
+      - pills--add
+      - pills--subtract
+      - pointer-text
+      - region-analysis--area
+      - region-analysis--volume
+      # registration
```
Comment (shixiedesign) [inline packages/icons/categories.yml:347]: # to -

File: packages/icons/categories.yml [hunk 5]
```diff
@@ -1,25 +1,764 @@
...
+      - chart--bar
+      - chart--bubble
+      - chart--line
+      - chart--line--data
+      - chart--network
+      - chart--pie
+      - chart--scatter
+      - chart--venn-diagram
+      - classifier--language
+      # concept
```
Comment (shixiedesign) [inline packages/icons/categories.yml:453]: another one

**Review by shixiedesign — COMMENTED** [review]
There are a lot of places where `-` is replaced with `#` . I noticed them because it's not the yml format. Is this intentional? Wouldn't they throw errors? Should they all be `-`?

File: packages/icons/categories.yml (hunk 1 above)
Comment (conradennis) [reply packages/icons/categories.yml:155]: these icons are not yet added to the repo but will be very soon so at @chrisconnors-ibm request I added # before them so they'd be commented out until they are in the repo

**Review by joshblack — APPROVED** [review]

**Comment by joshblack** [comment]
Commit message should be:

```
chore(icons): update categories.yml
```

**Review by shixiedesign — APPROVED** [review]
Cool! Thanks Conrad for explaining.

**Pull request merged**